contrib/mesos/docs/ha.md

```diff
@@ -0,0 +1,66 @@
+## High Availability
+
+### Scheduler
+
+Implementation of scheduler HA features has started and includes:
+
+- Checkpointing by default (`--checkpoint`)
+- Large failover-timeout by default (`--failover_timeout`)
+- Hot-failover w/ multiple scheduler instances (`--ha`)
+- Best effort task reconciliation on failover
+
+#### Multiple Instances
+
+Multiple scheduler instances may be run to support a hot-failover scenario in which one scheduler fails and another takes over immediately.
+At any moment in time only one scheduler is actually registered with the leading Mesos master.
+Scheduler leader election is implemented using etcd so it is important to have an HA etcd configuration established for reliable scheduler HA.
+
+It is currently recommended that no more than 2 scheduler instances be running at the same time.
```

Comment:
Just for my own curiosity, is there any reason why it wouldn't work, given that only one is active at a time?